### PR TITLE
ops: naming the return of some functions

### PR DESCRIPTION
**Pull request by btrb**
This PR is for renaming some variable names, and also for avoiding the return of unnamed results when declaring OPS Dats and OPS Args.

### CODE REVIEW

File: devito/ops/operator.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def _specialize_iet(self, iet, **kwargs):
             if f.is_Constant:
                 continue
 
-            pre_time_loop.extend(create_ops_dat(f, name_to_ops_dat, ops_block))
+            pre_time_loop.extend(list(create_ops_dat(f, name_to_ops_dat, ops_block)))
```
Comment (vmickus) [inline devito/ops/operator.py]: This is the only part that i didn't like, but for the sake of the rest of  the code, I think its better.

File: devito/ops/transformer.py [hunk 2]
```diff
@@ -191,13 +190,18 @@ def create_ops_dat(f, name_to_ops_dat, block):
...
+    d_m_val = Expression(ClusterizedEq(Eq(d_m, ListInitializer(d_m_val))))
 
-    return res
+    ops_dat = OpsDatDecl(dim_val=dim_val,
+                         base_val=base_val,
+                         d_p_val=d_p_val,
+                         d_m_val=d_m_val,
+                         ops_decl_dat=ops_decl_dat)
+
+    return ops_dat
```
Comment (vmickus) [inline devito/ops/transformer.py]: What do you about just returning the `OpsDatDecl`? 
The code sure is cleaner now.

**Comment by FabioLuporini** [comment]
I'm quite happy about this, I'll just wait for you guys, @vmickus and @maelso , to formally approve the PR here on GH, and then I'll proceed with merging it

File: devito/ops/transformer.py (hunk 2 above)
Comment (btrb) [reply devito/ops/transformer.py]: sure, thought about it as soon as pushed. thanks for pointing

**Comment by maelso** [comment]
Looks good.

File: devito/ops/transformer.py [hunk 3]
```diff
@@ -240,43 +241,56 @@ def create_ops_par_loop(trees, ops_kernel, parameters, block, name_to_ops_dat,
...
     )))
 
+    ops_args = []
+    for p in parameters:
+        ops_arg = create_ops_arg(p,
+                                 accessible_origin,
+                                 name_to_ops_dat,
+                                 par_to_ops_stencil)
+
+        ops_args.append(ops_arg.ops_type(ops_arg.ops_name,
```
[the remark this answers is not in the document]
Comment (btrb) [reply devito/ops/transformer.py]: indeed i haven't see a pattern like that before in the codebase
it is correct though. the thing here is that `ops_arg.ops_type` is a `[Function](https://github.com/opesci/devito/blob/master/devito/ops/utils.py#L23)`

**Comment by FabioLuporini** [comment]
merged.

**Pull request merged**